Sources/Networking/HTTPClient/ETagManager.swift

```diff
@@ -173,18 +193,22 @@ extension ETagManager {
         var eTag: String
         var statusCode: HTTPStatusCode
         var data: Data
+        @DefaultDecodable.Now
+        var lastUsed: Date
```

Comment:
What's this used for?

Reply:
You can see the full context here: https://linear.app/revenuecat/issue/SDK-3023/improve-load-shedder-performance-send-etag-validation-date

I know the code might not be the most clear, so suggestions for how to improve it are welcome.

Reply:
I think this is about as clear as it gets, but I'll add a docstring to the actual property so its purpose is clear to future maintainers.

Reply:
marking as resolved, @vegaro lmk if there are still any concerns with this part

Reply:
maybe it'd be clearer if it's also called "validation" rather than "used", for consistency with the rest of the code?